docs/package.json

```diff
@@ -2,9 +2,19 @@
   "name": "@cfpb/cfpb-design-system",
   "version": "1.0.0",
   "description": "CFPB Design System documentation website",
-  "homepage": "https://github.com/cfpb/design-system",
+  "homepage": "https://cfpb.github.io/design-system/",
   "license": "MIT",
-  "main": "dist/main.js",
```

Comment:
I removed this as there isn't a `dist/main.js` file. There is a `dist/js/design-system.js` file, should it be pointing to that?